metrics/falcon.go

```diff
@@ -0,0 +1,48 @@
+package metrics
+
+import falcon "github.com/niean/goperfcounter"
+
+type falconGauge struct {
+	name string
+}
+
+// Add add value of counter
+func (f *falconGauge) Add(value int64) {
+	falcon.SetCounterCount(f.name, value)
+}
+
+// Inc add value of counter, value = 1
+func (f *falconGauge) Inc() {
+	f.Add(1)
+}
+
+// Decrease decrease value of counter
+func (f *falconGauge) Decrease(value int64) {
```

Comment:
```suggestion
func (f *falconGauge) Dec(value int64) {
```

The naming style should be consistent, as you use `Inc` instead of "Increase".